### DISCUSSION TITLE
How to install offline manually

### DISCUSSION THREAD

**Discussion by kdalenberg** [harfbuzz/harfbuzz#4930] [Q&A]
I am attempting to install the r package "systemfonts" on a windows system that is not allowed out to the internet for security reasons.  When I attempt that install I get an error about downloading harfbuzz from github.  

I can download and extract the harfbuzz windows 64 bit zip file, but how do I install it on the windows system so the systemfonts r package doesn't try to download it?

**Comment by khaledhosny** [comment]
That is a question for the "systemfonts" r package, I don’t think anyone here has an answer to this question.
